pkg/db/src/implementations/version-1.2/books.ts

```diff
@@ -37,10 +37,29 @@ class Books implements BooksInterface {
 	}
 
 	async get(isbns: string[]): Promise<(BookEntry | undefined)[]> {
-		const rawBooks = await this.#db._pouch.allDocs<BookEntry>({ keys: isbns.map((isbn) => `books/${isbn}`), include_docs: true });
+		const rawBooks = await this.#db._pouch
+			.allDocs<BookEntry>({ keys: isbns.map((isbn) => `books/${isbn}`), include_docs: true })
+			.then((docs) => unwrapDocs(docs));
```

Comment:
Just move the comment from below to above the `.then...` (as it describes that part)